Makefile

```diff
@@ -98,13 +98,11 @@ clean-launch:
 integration-tests:
 	go test ./src/server -timeout 120s
 
+protofix:
+	docker build -t pachyderm/protofix -f Dockerfile.protofix .
+
 proto:
-	go get -v go.pedge.io/protoeasy/cmd/protoeasy
-	rm -rf src/server/vendor
-	sudo -E protoeasy --grpc --grpc-gateway --go --go-import-path github.com/pachyderm/pachyderm/src src
-	go install github.com/pachyderm/pachyderm/src/server/cmd/protofix
-	protofix fix src
-	git checkout src/server/vendor
+	docker run -v $(PWD):/go/src/github.com/pachyderm/pachyderm pachyderm/protofix 
```

Comment:
We should have 2 versions of this command, `make proto` and `make docker-proto` so that the other version can still be used. I think people on macs are likely to need the old version of the command for a while.